src/TestProjects.ruleset

```diff
@@ -1,6 +1,5 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The file encoding of this file was changed to `UTF with signature` by the migration.